source/gems/dry-logic/predicates.html.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Predicates
+layout: gem-single
+---
+
+Dry-logic comes with a lot predicates to compose multiple rules:
+
+``` ruby
+require 'dry/logic'
+require 'dry/logic/predicates'
+
+include Dry::Logic
+```
+
+Now you can access all built-in predicates:
+
+``` ruby
+Predicates[:key?]
+=> #<Method: Module(Dry::Logic::Predicates::Methods)#key?>
+```
+
+In the end predicates return a called method:
```

Comment:
Maybe a better way to put it would be `In the end predicates return true or false`